src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,13 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class ParserTest {
+
+    @Test
+    void test() {
+        assertEquals(2, 2);
```

Comment:
I think this test is incredibly effective in ascertaining that 2 is indeed equal to 2.